sshmux/ssh.py

```diff
@@ -83,7 +87,7 @@ def validate_user(ctx, param, value):
     return value
 
 
-def validate_key(value):
+def validate_key(ctx, param, value):
```

Comment:
Using it this way will make private key mandatory. Even if you give password, it will expect a valid path of key file. can you test this.